src/openforms/js/components/formio_builder/index.js

```diff
@@ -14,6 +14,7 @@ onLoaded(() => {
   [...FORM_BUILDERS].forEach(node => {
     const configurationInput = BEM.getChildBEMNode(node, BLOCK_FORM_BUILDER, INPUT_ELEMENT);
     const configuration = JSON.parse(configurationInput.value) || {display: 'form'};
+    // FIXME: handle translations?
```

Comment:
TODO: should be possible to manage component translations from standalone admin page too